src/components/ViewPlan.js

```diff
@@ -70,6 +70,26 @@ export default class ViewPlan extends React.Component {
         }
       }
 
+      // get user name
+      url = `/api/user/${userId}`;
+      response = await fetch(url);
+      if (response.ok) {
+        // get data from the response
+        obj = await response.json();
+        this.setState({
+          studentName: obj.firstName + " " + obj.lastName
+        });
+      } else {
+        // we got a bad status code
+        try {
+          throw response;
+        } catch (err) {
+          err.text().then(errorMessage => {
+            alert(errorMessage);
+          });
+        }
+      }
+
```

Comment:
The alert "no plan was found" was given 3 times...I'm not sure if this is the part of the code that executes that but we should probably look into it

Reply:
I think that is due to changes I made in merge and not here.